.github/workflows/rc_sync.yml

```diff
@@ -52,16 +53,17 @@ jobs:
           # Exclude some files known to be divergent during the release process from the PR
           git checkout master -- pennylane/_version.py
           git checkout master -- doc/releases/changelog-dev.md
-          if ! git diff-index --quiet HEAD; then
-            git config user.name "GitHub Actions Bot"
-            git config user.email "<>"
-            git commit -m "exclude files from pr"
+          git config user.name "GitHub Actions Bot"
+          git config user.email "<>"
+          git commit -m "exclude files from pr"
+          if ! git diff master HEAD --quiet -- $(git diff --name-only master...); then
```

Comment:
It's worth leaving an explanation on this line. Later on, it will be helpful to have a description.

Reply:
done